Finmer.Game/Gameplay/SceneCombat2.cs

```diff
@@ -532,6 +543,12 @@ private void ShowUIDefault(GameUI ui)
                     Label = "Fight",
                     Tooltip = "Attack another character to deal damage."
                 });
+            }
+
+            //Grapple
```

Comment:
On a separate note: please do make sure to stick to the surrounding style - there should be space between the comment and the `//` (check CONTRIBUTING.md for more info). I know it's pedantic but consistency really helps readability.